libs/ui/organisms/src/lib/AuctionHeader/AuctionHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { Icon, IconType, Row } from '@inheartive/ui/atoms';
+import { Link } from 'react-router-native';
+import { theme } from 'native-base';
+
+const styles = StyleSheet.create({
```

Comment:
Each time a different type of styling is added, here with StyleSheet.create, then with style inline, then styling components. It would be good to stick to one system and agree on it in a team.